dev-util/rt-tests/rt-tests-1.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
Old copyright header, you can use repoman to commit so it fixes these things for you automatically.